cartridge/issues.lua

```diff
@@ -178,6 +181,22 @@ local function list_on_instance()
         })
     end
 
+    if failover.consistency_needed()
+    and failover.get_active_leaders()[replicaset_uuid] == instance_uuid
```

Comment:
Seems some comment could be useful here. I don't get when all this conditions is true simultaneously.

Reply:
Done